app/views/lead_providers/guidance/npq_usage.html.erb

```diff
@@ -45,68 +48,151 @@
-</h2>
+</h3>
+
 <p class="govuk-body-m">
   This scenario begins after a participant has been added to the service by registering for an NPQ course
 </p>
-<h3 class="govuk-heading-m">1. Provider accepts an NPQ application</h3>
+
+<h4 class="govuk-heading-s">
+  1. Provider accepts an NPQ application
+</h4>
+
 <p class="govuk-body-m">You can accept someone if they have successfully completed all application steps ahead of starting their course.</p>
+
 <p class="govuk-body-m">
   <code>POST /api/v1/npq-applications/{id}/accept</code>
 </p>
+
 <p class="govuk-body-m">This returns an
   <a href="/lead-providers/guidance/reference#npqapplicationresponse-object" class="govuk-link">NPQ application record</a>.
 </p>
 
-<h2 id="reject-npq-application" class="govuk-heading-l">
+<h3 id="reject-npq-application" class="govuk-heading-m">
   Reject an NPQ application
-</h2>
+</h3>
+
 <p class="govuk-body-m">
   This scenario begins after a participant has been added to the service by registering for an NPQ course
 </p>
-<h3 class="govuk-heading-m">1. Provider rejects an NPQ application</h3>
+
+<h4 class="govuk-heading-s">
+  1. Provider rejects an NPQ application
+</h4>
+
 <p class="govuk-body-m">You will need to reject someone if you will be unable to train them. For example they have failed the rest of their application, have decided against studying an NPQ or are unable to secure funding.</p>
+
 <p class="govuk-body-m">
   <code>POST /api/v1/npq-applications/{id}/reject</code>
 </p>
+
 <p class="govuk-body-m">This returns an
   <a href="/lead-providers/guidance/reference#npqapplicationresponse-object" class="govuk-link">NPQ application record</a>.
 </p>
 
+<h3 id="handling-deferrals" class="govuk-heading-m">
+  Handling deferrals
+</h3>
+
+<p class="govuk-body-m">
+  If a participant wishes to defer you can accept the participant to show they are enrolled. However only send the start declaration once the participant has started.
+</p>
+
+<h3 id="handling-changes-circumstances" class="govuk-heading-m">
+  Handling applications with changes in circumstances
+</h3>
+
+<p class="govuk-body-m">
+  There are many possible reasons why there might be a change in circumstances of an application. These may include:
+</p>
+
+<ul class="govuk-list govuk-list--bullet">
+  <li>participant selected incorrect course during their application</li>
+  <li>participant made a mistake during their application</li>
+  <li>participant now wishes to take another course instead</li>
+  <li>participant now wishes to fund their NPQ differently</li>
+</ul>
+
+<p class="govuk-body-m">
+  If there is a mistake in the application and does not truly reflect reality for example where a participant registers for an NPQ programme but wishes to change the programme after registration. The provider should reject that participant and ask them to re-register on the NPQ registration service and enter the correct details. Once the new application is available you can then accept.
```

Comment:
The first sentence here doesn't quite read properly.

Reply:
I've tweaked it somewhat. Please have another read